numpy/tests/test_public_api.py

```diff
@@ -271,19 +271,12 @@ def test_NPY_NO_EXPORT():
 
 def is_unexpected(name):
     """Check if this needs to be considered."""
-    if '._' in name or '.tests' in name or '.setup' in name:
-        return False
-
-    if name in PUBLIC_MODULES:
-        return False
-
-    if name in PUBLIC_ALIASED_MODULES:
-        return False
-
-    if name in PRIVATE_BUT_PRESENT_MODULES:
-        return False
-
-    return True
+    return (
+        '._' not in name and '.tests' not in name and '.setup' not in name
+        and name not in PUBLIC_MODULES
+        and name not in PUBLIC_ALIASED_MODULES
+        and name not in PRIVATE_BUT_PRESENT_MODULES
+    )
```

Comment:
I like the breaking across lines, but putting a lot of computation in the return is, I think, a bad habit to be avoided.

Reply:
The computation could be put in a variable, and then the variable returned. But then a `RET` rule would suggest to avoid the variable 😀

Reply:
I've kept this change because it's hardly a computation. I could use a temporary variable, but that's probably overkill.